Prox/Prox/Models/ReviewProvider.swift

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Firebase
+
+class ReviewProvider {
+
+    let url: String
+
+    let rating: Double
+    let reviews: [String]
+    let totalReviewCount: Int
```

Comment:
I made `url` required because I figured we really need to be able to attribute the reviews and, if we can't, it's better that we say, "Data not available" than show data that's linkable.

That being said, users may prefer to see the data even if we can't link directly, in which case, I suppose we could just change it.